index.dd

```diff
@@ -240,6 +240,34 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE_RUNNABLE Generate a config parser at Compile-Time,
+----
+import std.algorithm, std.conv, std.range, std.stdio, std.traits;
+struct Config {
+    int a, b;
+    string c;
+}
+void main() {
+    Config conf;
+    // use the generated parser for each entry
```

Comment:
Using "generated parser" but there is nothing called "parser" in the example.  Perhaps rename the `read` function below to `parser`.

Reply:
Fair point - I renamed it to `parse` as then `"b=1,a=2,c=hello".splitter(",").each!(e => conf.parse(e));` reads nicer